pages/standard/clinical.tsx

```diff
@@ -1,51 +1,32 @@
 import React from 'react';
-import HtanNavbar from '../../components/HtanNavbar';
-import Footer from '../../components/Footer';
-import Container from 'react-bootstrap/Container';
-import Row from 'react-bootstrap/Row';
-import Breadcrumb from 'react-bootstrap/Breadcrumb';
-import { GetServerSideProps, GetStaticProps } from 'next';
-import fetch from 'node-fetch';
-import { CmsData } from '../../types';
-import { WORDPRESS_BASE_URL } from '../../ApiUtil';
+import { GetStaticProps } from 'next';
 
-export interface CdsProps {
-    data: CmsData[];
-}
+import DataStandard, { DataStandardProps } from "../../components/DataStandard";
+import { getStaticContent } from '../../ApiUtil';
+import { getDataSchema } from "../../lib/dataSchemaHelpers";
 
-function Cds(data: CdsProps) {
+const Cds: React.FunctionComponent<DataStandardProps> = props => {
     return (
-        <>
-            <HtanNavbar />
-            <Container>
-                <Row>
-                    <Breadcrumb className="mt-3">
-                        <Breadcrumb.Item href="/">Home</Breadcrumb.Item>
-                        <Breadcrumb.Item href="/standards">
-                            Data Standards
-                        </Breadcrumb.Item>
-                        <Breadcrumb.Item active>Clinical Data</Breadcrumb.Item>
-                    </Breadcrumb>
-                </Row>
-                <Row>
-                    <span
-                        dangerouslySetInnerHTML={{
-                            __html: data.data[0].content.rendered,
-                        }}
-                    ></span>
-                </Row>
-            </Container>
-            <Footer />
-        </>
+        <DataStandard {...props} title="Clinical Data" />
     );
 }
 
 export const getStaticProps: GetStaticProps = async (context) => {
-    let slugs = ['data-standards-cds-blurb'];
-    let overviewURL = `${WORDPRESS_BASE_URL}${JSON.stringify(slugs)}`;
-    let res = await fetch(overviewURL);
-    let data = await res.json();
-    return { props: { data } };
+    const data = await getStaticContent(['data-standards-cds-blurb']);
+
+    // TODO this may not be the complete list of clinical data
+    const { dataSchemaData, schemaDataMap } = await getDataSchema([
+        "bts:Demographics",
+        "bts:Diagnosis",
+        "bts:Exposure",
+        "bts:FamilyHistory",
+        "bts:FollowUp",
+        "bts:MolecularTest",
+        "bts:Treatment",
+        "bts:ClinicalDataTier2"
```

Comment:
@inodb This may not be the complete list of clinical data attributes. We might be missing some here.

Reply:
gotcha - maybe this is ok for now